core/scripts/utils/ssr-redirect.ts

```diff
@@ -85,13 +85,12 @@ export function createRedirectProxy (context: Context, expressResponse: Response
   // returns proxy for Express Reponse object
```

Comment:
Reponse in comment! Fix it if you want, PR looks great right now.

Reply:
done